### PR TITLE
Fix/remove input shadow

### LINKED ISSUE #911: [BUG] Remove shadow and change text on disabled input
**Describe the bug**
1. Right now there is an elevation on our disabled input-fields, which indicate that they are clickable - But they are not. 

2. Also the text color is "active" right now, where it should be a more toned down color (semi-dark)

**To Reproduce**

1. Go to https://cookbook.kirby.design/home/showcase/form-field
2. See error

**Expected behavior**
The elevation should be removed when the field is disabled: https://zpl.io/VDwPqre and the color on the text should be semi-dark

**Screenshots**

**Please complete the following information:**

- OS: [e.g. Windows, MacOS, iOS]
- Browser [e.g. Chrome, Safari]

**Additional context**

### PR DESCRIPTION
**Pull request by flemcito**
This PR closes #911 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] cookbook application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Disabled input fields have elevation and high contrast text color.
 
Issue Number: N/A

## What is the new behavior?
The box-shadow is removed when disabled (no elevation), and the text color is changed to semi-dark

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts [hunk 1]
```diff
@@ -58,6 +59,16 @@ describe('FormFieldComponent', () => {
     });
   });
 
+  fdescribe('When disabled', () => {
```
Comment (peterlindholm) [inline libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts:62]: Remove 'f'

```suggestion
  describe('When disabled', () => {
```

**Review by jakobe — CHANGES_REQUESTED** [review]
Nice style change 👍 
Just one small change to test, as it's currently a false negative ;)

File: libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts [hunk 2]
```diff
@@ -58,6 +59,16 @@ describe('FormFieldComponent', () => {
...
+    it('should not have elevation', () => {
+      spectator = createHost(`<kirby-form-field>
+        <input kirby-input disabled value="Disabled input" />
+      </kirby-form-field>`);
+
+      expect(spectator.element).toHaveComputedStyle({ 'box-shadow': 'none' });
```
[the remark this answers is not in the document]
Comment (flemcito) [reply libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts]: Makes a ton of sense, fixed :)

File: libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts (hunk 1 above)
Comment (flemcito) [reply libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts]: Thanks 👍

File: libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts [hunk 3]
```diff
@@ -49,6 +46,10 @@ describe('FormFieldComponent', () => {
     it('should have correct margin', () => {
       expect(spectator.element).toHaveComputedStyle({ 'margin-bottom': size('s') });
     });
+    it('should have elevation', () => {
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts:49]: Consider readability:
```suggestion

    it('should have elevation', () => {
```

File: libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts [hunk 4]
```diff
@@ -59,13 +60,13 @@ describe('FormFieldComponent', () => {
     });
   });
 
-  fdescribe('When disabled', () => {
+  describe('When disabled', () => {
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts:63]: Align casing with existing tests:
```suggestion
  describe('when disabled', () => {
```

File: libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts [hunk 5]
```diff
@@ -48,6 +46,10 @@ describe('FormFieldComponent', () => {
     it('should have correct margin', () => {
       expect(spectator.element).toHaveComputedStyle({ 'margin-bottom': size('s') });
     });
+    it('should have elevation', () => {
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts:49]: Consider moving to describe block `with slotted input` and adding similar test to `with slotted textarea` block...

**Review by jakobe — CHANGES_REQUESTED** [review]
Consider adding 2 additional tests, that tests the box-shadow on an enabled input and textarea...

File: libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts (hunk 3 above)
Comment (flemcito) [reply libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts]: Moved (and added spacing)

File: libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts (hunk 5 above)
Comment (flemcito) [reply libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts]: Makes sense 👍  
Done

File: libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts (hunk 4 above)
Comment (flemcito) [reply libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts]: Done

**Review by jakobe — APPROVED** [review]
🎉

**Pull request merged**